src/circllhist.c

```diff
@@ -1115,6 +1115,8 @@ void
 hist_clear(histogram_t *hist) {
   int i;
   ASSERT_GOOD_HIST(hist);
+  hist->used = 0;
+  // just to be sure, clear the counts as well
   for(i=0;i<hist->used;i++)
```

Comment:
This for loop has no purpose if you set it to 0 there.

Reply:
Good point. But it might be worth clearing the counts of the previously used buckets, no?

So I would move the assignment behind the loop.